common/src/main/webapp/less/variables-dark.less

```diff
@@ -282,6 +282,7 @@
   --checkbox-focus-background: @gray-dark;
   --checkbox-active-background: var(--control-background-color);
   --checkbox-active-focus-background: var(--control-background-color);
+  --checkbox-active-focus-check-color: @selectedTextColor;
```

Comment:
Would you use a variable here too?

Reply:
I don't know. There is currently a mix of `@` and `var`. I'm not really sure when it's worth creating a new `var` from an `@`.